Kernel/API/Unveil.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+
+namespace Kernel {
+
+#define UNVEIL_CURRENT_PROGRAM (1 << 0)
```

Comment:
Could this use enum class or a constexpr instead? Or coud this follow this style, for example:

```cpp
enum class InodeWatcherFlags : u32 {
    None = 0,
    Nonblock = 1 << 0,
    CloseOnExec = 1 << 1,
};

AK_ENUM_BITWISE_OPERATORS(InodeWatcherFlags);
```